sys/include/sched_round_robin.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2020 TUBA Freiberg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    sche_round_robin Round Robin Scheduler
+ * @ingroup     sys
+ * @brief       This module module provides round robin scheduling within
+ *              for all runnable threads within each not masked priority.
+ *              Priority 0 is masked by default.
+ *              There is no fainess involved in this scheuler. It schedules,
+ *              when the timer ticks. Keep it as simple as possible.
+ *
+ *
+ * @{
+ *
+ * @file
+ * @brief       Round Robin Scheduler
+ *
+ * @author      Karl Fessel <karl.fessel@ovgu.de>
+ *
+ */
+#ifndef SCHED_ROUND_ROBIN_H
+#define SCHED_ROUND_ROBIN_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#if !defined(SCHED_RR_TIMEOUT) || defined(DOXYGEN)
+/**
+ * @brief Time between round robin calls in µs
+ *
+ * @details Defaults to 1ms
+ */
+#define SCHED_RR_TIMEOUT 1000
+#endif
+
+#if !defined(SCHED_RR_MASK) || defined(DOXYGEN)
+/**
+ * @brief masks off priorities that should not be scheduled default: 0 is masked
+ */
+#define SCHED_RR_MASK (1 << 0)
+#endif
+
+/**
+ *  @brief  Registers the runqueue change callback and initialises round robin
+ */
+void init_sched_round_robin(void);
```

Comment:
`sched_round_robin_init()`

Reply:
when i changed the name in auto_init.c i saw where i got the name from `init_schedstatistics(void)`